sdk/formrecognizer/azure-ai-formrecognizer/samples/sample_train_model_with_labels.py

```diff
@@ -10,11 +10,16 @@
 FILE: sample_train_model_with_labels.py
```

Comment:
Hopefully adding these forms is fine and it doesn't add up to the repo size as much.

Reply:
It was a lot worse when I had ALL the training files in the repo before we moved to the shared storage account 😆